torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -128,6 +137,28 @@ def __init__(self):
         self.comm_registry.add(torch.ops._dtensor.shard_dim_alltoall)
         self.advanced_module_tracker = ModuleParamaterShardingTracker()
 
+    def generate_module_tracing_table(self):
```

Comment:
Maybe call it `generate_module_tracing_table` to explicitly say that there will be prints from this function?

Reply:
agree. we'd better have this function return a string and let user print it when they want. Besides, we can have a api to dump this to file as a follow up.